webui/react/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "engines": {
-    "node": ">=18.16.0 <19",
+    "node": ">=20.1.0 <21",
     "npm": ">=9.5.1"
```

Comment:
Could you change it to the minimum npm version

Reply:
I've updated the npm version and settled on playwright 1.34 (not 1.39); With firefox and webkit passing, it looks like this may be ready to go